packages/web/components/Patron/PatronTile.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+  Avatar,
+  Box,
+  Heading,
+  HStack,
+  MetaTag,
+  MetaTile,
+  MetaTileBody,
+  MetaTileHeader,
+  Text,
+  VStack,
+  Wrap,
+  WrapItem,
+} from '@metafam/ds';
+import { MetaLink } from 'components/Link';
+import { PlayerContacts } from 'components/Player/PlayerContacts';
+import { PlayerTileMemberships } from 'components/Player/PlayerTileMemberships';
+import { utils } from 'ethers';
+import { PlayerFragmentFragment } from 'graphql/autogen/types';
+import React from 'react';
+import {
+  getPlayerCoverImage,
+  getPlayerImage,
+  getPlayerName,
+} from 'utils/playerHelpers';
+
+type Props = {
+  player: PlayerFragmentFragment;
+};
+
+export const PatronTile: React.FC<Props> = ({ player }) => {
+  return (
+    <MetaTile key={player.id}>
```

Comment:
I don't think you need to put `key` again here, the one on PatronList l.13 is enough for React, if I'm not wrong